frontend/compositions/header/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react"
+import NPDCLogo from './assets/NPDCLogo.svg'
+import styles from './header.module.css'
+
+
+export default function Header() {
+  let [selected, setSelected] = useState('Search');
+
+  const handleNavChange = (e: any) => {
```

Comment:
I believe we want to avoid `any` as a type - I believe the type you'll need here is `MouseEvent<HTMLAnchorElement>` (once we add the links at least - and `MouseEvent` comes from react)

Reply:
> Hey! I can't seem to find your branch to pull - but the screenshot makes it seem like the page tabs and donate button are about halfway across the page - design-wise we intended them to be on the right side of the page (apologies if that wasn't clear in the wireframes!

Hi Sam, thank you for all the helpful feedback! 

I'll push up my latest changes tonight, with a change to the grid layout so that the page tabs and donate button stay on the right side. The wireframes are totally clear - I was incorrectly setting up my columns, and will make sure to attach screenshots of the fix!